src/clr-angular/data/datagrid/datagrid-column-separator.ts

```diff
@@ -61,6 +61,7 @@ export class ClrDatagridColumnSeparator {
 
   private redFlagTracker(resizeTrackerEl: HTMLElement) {
     let isWithinMaxResizeRange: boolean;
+    // @TODO(JEREMY) Review this, it will always be true because above is always null
     if (isWithinMaxResizeRange !== this.columnResizerService.isWithinMaxResizeRange) {
```

Comment:
Hee sorry, overlooked this one. it should have been a property instead of a variable. It will work but we are setting style through `Renderer2` constantly during resizing.

Reply:
I figured it was meant to be a property, but didn't spend the time to dive deeper.